herbert_core/classes/MPIFramework/@ClusterParpoolWrapper/ClusterParpoolWrapper.m

```diff
@@ -137,7 +137,11 @@
             % variables, but if the cluster is remote, the envriomental
             % variables transfer should be investigated
             obj.set_env();
-            h_worker = str2func(obj.worker_name_);
+            if obj.is_compiled_script_
+                h_worker = @worker_v2;
+            else
```

Comment:
Right... yes, I had misunderstood how it worked... so if you compile Matlab with the parallel toolbox it will spawn additional processes for `parpool` without needing a compiled worker... which is why it need the reference to the worker function. I'll remove this and make it an error (via an `assert` to initialise a `@ClusterParpoolWrapper` with a compiled worker).